code/style.css

```diff
@@ -1,4 +1,359 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color:grey;
 }
+
+/*header layout*/
+.header {
+  background-image: url("https://images.unsplash.com/photo-1497105943914-b90d55145023?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1270&q=80");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: auto;
+  }
+  
+  .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
```

Comment:
Check indentation of your CSS file, only the properties should be indented here, not the whole element :)